.travis.yml

```diff
@@ -62,7 +41,7 @@ deploy:
     on:
       repo: dart-lang/site-www
       branch: master
-      condition: $TASK == *build*
+      condition: $TASK == *build* && $TRAVIS_DART_VERSION == stable
```

Comment:
We might consider making the wildcard more exact. It's possible that more scripts with the world "build" might be introduced. Maybe *tool/build.sh* would be less prone to breakage?

Reply:
Good idea. Done